indico/modules/rb/forms/settings.py

```diff
@@ -42,6 +42,17 @@ class SettingsForm(IndicoForm):
     notification_before_days_monthly = IntegerField(_('Send booking reminders X days before (monthly)'),
                                                     [InputRequired(), NumberRange(min=1, max=30)])
     notifications_enabled = BooleanField(_('Reminders enabled'))
+    end_notification = IntegerField(_('Send reminders about ending bookings X days before their end date '
+                                      '(daily)'),
```

Comment:
I think this is very confusing. Maybe we should say "(daily bookings)" instead of just "(daily)'? Otherwise people may think that the reminders will be sent daily.

Reply:
Yeah, I will change it soon when I will move those settings to the new RB.